test/quantum/executor_test.exs

```diff
@@ -408,7 +409,7 @@ defmodule Quantum.ExecutorTest do
           assert :ok == wait_for_termination(task)
         end)
 
-      assert logs =~ ~r/\(exit\) :failure/
+      assert logs =~ ~r/type=exit value=failure/
```

Comment:
I think it would make sense to split this into two assertions (one for `type`, one for `value`) to make sure the tests pass.